breathecode/payments/views.py

```diff
@@ -967,7 +967,6 @@ class AcademySubscriptionView(APIView):
 
     extensions = APIViewExtensions(sort="-id", paginate=True)
 
-    @capable_of("read_subscription")
```

Comment:
Academy views requires `capable_of`, add `read_subscription` in `create_academy_roles.py` if not added